src/Symfony/Component/Intl/Data/Bundle/Reader/IntlBundleReader.php

```diff
@@ -34,7 +34,6 @@ public function read($path, $locale)
             // Never enable fallback. We want to know if a bundle cannot be found
             $bundle = new \ResourceBundle($locale, $path, false);
         } catch (\Exception $e) {
-            // HHVM compatibility: constructor throws on invalid resource
```

Comment:
There is a config in Intl, that translate errors into exceptions. Before removing the try-catch, this has to be tested with that intl config `intl.use_exceptions`.

http://php.net/manual/de/intl.configuration.php